vkcs/internal/services/networking/neutron_errors.go

```diff
@@ -0,0 +1,22 @@
+package networking
+
+import (
+	"encoding/json"
+)
+
```

Comment:
It also would better to add constants defining some errors instead of using plain string literals.

Reply:
done